client/src/pages/Property.tsx

```diff
@@ -1,17 +1,43 @@
-import { Container } from "@chakra-ui/react";
-import { Navbar } from "components/nav";
-import React from "react";
+import React, { useState } from "react";
 import InfoCard from "components/InfoCard";
-import PropertyCard from "components/PropertyCard";
-import Review from "components/review";
-import SearchBar from "components/searchBar";
-const Property = () => {
+import Review from "components/Review";
+import { useLocation } from "react-router-dom";
+import { props } from "./../components/Posting";
+import { Comment } from "./../components/interfaces/Comment";
```

Comment:
```suggestion
import { props } from "components/Posting";
import { Comment } from "components/interfaces/Comment";
```
I enabled absolute import so the above should work